contactaccessrights/js/test/controllers/access-rights-modal-ctrl_test.js

```diff
@@ -222,6 +222,45 @@ define([
             expect(ctrl.errorMsg.length).not.toBe(0);
           });
         });
+
+        describe("when location changes", function () {
+          it("cannot submit without change", function () {
+            expect(ctrl.canSubmit).toBe(false);
+          });
+
+          it("can submit when location and region changes", function () {
+
+            ctrl.selectedData.locations.push({
+              name: "Home",
+              label: "Home"
+            });
+
+            ctrl.selectedData.regions.push({
+              name: "Home",
+              label: "Home"
+            });
+
+            ctrl.selection();
+
+            expect(ctrl.canSubmit).toBe(true);
+          });
+
+
+        });
+
+        describe('prevent multiple save', function () {
+          beforeEach(function () {
+            ctrl.submit();
+          });
+
+          it('submitting should be true', function () {
+            expect(ctrl.submitting).toBe(true);
+          });
+
+          xit('submitting should be false when promise has finished', function () {
+            // TODO
```

Comment:
No need for this test case, If it resolves or rejects the variable will be reinitialized.